1-estrategia/documentar.md

```diff
@@ -0,0 +1,93 @@
+* Funcionalidad: se refiere a que todo lo disponible en la web funcione correctamente.
+* Findability: se refiere a la facilidad o dificultad que el usuario pueda tener para encontrar lo que busca.
+* Utilidad: hace referencia al uso que el usuario pueda dar a la web.
+* Estética: esta cualidad busca una apariencia atractiva para ofrecer al usuarui. La elección de la tipografía es un ejemplo de criterio influenciado por este factor.
+* Credibilidad: no es un factor puramente de diseño, pero las facilidades de uso que ofrezca la web al usuario, hará que ésta aparetemente sea más fiable y profesional.
+
+Si tomamos como ejemplo la homepage de Twitter, podemos concluir que la red social se puede basar, entre otros muchos, en los siguientes criterios de diseño:
+! [Twitter](http://allthingsd.com/files/2013/10/Twitter-Homepage-New-640x341.png)
+* Un uso adecuado de colores: el predominio del color azul hace referencia a los tonos de la identidad visual corporativa de la empresa.
+* Poca congestión de información: Twitter opta por una página principal con espacio suficiente. Así es más legible.
+* Una arquitectura predecible: tanto para los usuarios nuevos como para los que tienen cierta experiencia, la red social no tiene complicación alguna para su uso. El inicio de sesión o el registro son fácilmente identificables.
+* Sin distracciones: no existe ningún factor que distraiga al usuario. Está claro que hay que hacer en esa página.
+
+## Metricas de exito 
+Las métricas de éxito se utilizan como indicadores que permiten comprobar si las decisiones que hemos tomado en el proceso estratégico son las adecuadas para conseguir los objetivos que nos hemos planteado. Para ello, existen indicadores estándar, sobre los cuales deben partir los indicadores que vayamos a establecer posteriormente. Es recomendable a su vez, comenzar por preguntas o aspectos más generales para posteriormente, terminar en detalles más concretos. 
+ 
+Entre los criterios más comunes encontramos el denominado “HEART” Metric, cuyo nombre deriva de los acrónimos de cada uno de los aspectos que, según este criterio, se consideran más importantes. 
+
+1. Satisfacción de los usuarios en relación al producto  
+
+2. Grado de respuesta del usuario sobre el producto
+
+3. El número de usuarios que se registra en un determinado espacio temporal
+
+4. El número de usuarios que utiliza un producto durante un determinado tiempo    
+
+5. Grado de satisfacción respecto a la consecución de las tareas que permite la web
+
+Por su parte, el blog UX for masses nos ofrece algunas claves de cuáles pueden ser los aspectos más relevantes a la hora de medir los criterios según los cuáles los usuarios tendrán una mayor actividad en nuestro sitio web. En este sentido, nos ofrece la posibilidad de plantear preguntas a modo de cuestionario a los posibles usuarios, de tal forma que la opinión que recojamos de los propios usuarios podrá servirnos para realizar las mejoras que creamos pertinentes. 
```

Comment:
_el blog UX for masses_ no es una manera correcta de referenciar una fuente. Este artículo tiene un autor con nombre y apellido, y una fecha de publicación, que deberíais usar para referenciar e incluir en el apartado de Bibliografía al final.